bucket/smplayer.json

```diff
@@ -0,0 +1,52 @@
+{
+    "version": "22.2.0.0",
+    "description": "A video player forked from MPV. It can also play Youtube videos, search and download subtitles, and includes other features like a thumbnail generator and audio and video filters.",
+    "homepage": "https://www.smplayer.info/",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.fosshub.com/SMPlayer.html?dwl=smplayer-portable-22.2.0.0-x64.7z",
+            "hash": "c054f61eddc3ac97b0ce03c3001e6e080a9e6543de999a071df17e2a7fe3618d"
+        },
+        "32bit": {
+            "url": "https://www.fosshub.com/SMPlayer.html?dwl=smplayer-portable-22.2.0.0-win32-qt5.6.7z",
+            "hash": "be10cdc34a9b6282e1e91e9b789825a11939576750f1d5be839e2a4892532402"
+        }
+    },
+    "pre_install": [
+        "Rename-Item \"$dir\\SMPlayer.html\" 'dl.7z'",
+        "Expand-7zipArchive \"$dir\\dl.7z\" \"$dir\" -ExtractDir 'smplayer-portable' -Removal | Out-Null",
+        "'hdpi.ini', 'playlist.ini', 'favorites.m3u8', 'radio.m3u8', 'tv.m3u8' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
+        "}"
+    ],
+    "shortcuts": [
+        [
+            "smplayer.exe",
+            "SMPlayer"
+        ]
+    ],
+    "persist": [
+        "screenshots",
+        "hdpi.ini",
+        "playlist.ini",
+        "smplayer.ini",
```

Comment:
Can't get settings to persist after uninstallation - settings save to smplayer.ini but it's not linked to the smplayer.ini in the persist folder. Will we have to do something similar to strokesplus.net? Such as copying the file to persist manually in pre_uninstall: https://github.com/ScoopInstaller/Extras/pull/8703/files#diff-ea8096c8380d8ac840470e604fd0ddbcb123178f5deabae8d768ffc3277d5148R20-R23

Reply:
Yeah. We should definitely do that. Thank you for the details.

By the way, Copy-Item does not print output to screen by default.
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/copy-item?view=powershell-7.2
> -PassThru
Returns an object that represents the item with which you're working. By default, this cmdlet doesn't generate any output.

Reply:
Great, it works now! And thanks for the info, I'll remove the Out-Null from strokesplus.net now

Reply:
Oh I see you've already fixed it, thank you!